x-pack/plugins/endpoint/public/applications/endpoint/view/hosts/details/index.tsx

```diff
@@ -136,7 +142,15 @@ const PolicyResponseFlyoutPanel = memo<{
         </EuiTitle>
       </FlyoutSubHeader>
       <EuiFlyoutBody data-test-subj="hostDetailsPolicyResponseFlyoutBody">
-        <PolicyResponse />
+        {responseConfig !== undefined && responseActionStatus !== undefined ? (
+          <PolicyResponse
+            responseConfig={responseConfig}
+            responseActionStatus={responseActionStatus}
+            responseAttentionCount={responseAttentionCount}
+          />
+        ) : (
+          <p>No Policy Response Available</p>
```

Comment:
Need to i18n this. I'm ok if you do it in a subsequent PR.
Also - when we have a review with @bfishel , lets find out if perhaps we should use the `EuiCallout` or the `EuiEmptyPrompt` component to show a message here.

Reply:
sounds good i'm just using this as a placeholder for now.